packages/form/src/cli.ts

```diff
@@ -58,14 +125,20 @@ async function create() {
 		});
 	}
 
-	const relativeOutputPath = join(relativeOutputDir, "Form.tsx");
+	const relativeOutputPath = join(
+		relativeOutputDir,
+		toUseFramework.templateDir,
+	);
 	const outputPath = resolve(process.cwd(), relativeOutputPath);
 
-	const reactFormTemplate = await readFile(
-		new URL("../templates/react/Form.tsx", import.meta.url),
-	);
+	const templatePath = resolve(`./templates/${toUseFramework.templateDir}`);
 
-	await writeFile(outputPath, reactFormTemplate);
+	await copy(templatePath, outputPath, {
+		filter: (src) => {
+			const fileName = src.split("/").at(-1);
+			return !!fileName && !internalFiles.includes(fileName);
+		},
+	});
 
 	outro(`${bold(cyan(relativeOutputPath))} created. You're all set!`);
```

Comment:
```suggestion
	outro(`Form created. You're all set!`);
```

Including the output path doesn't make as much sense now that templates may contain multiple files. Just simplifying!